src/Event/EventManager.php

```diff
@@ -361,6 +369,7 @@ public function dispatch($event)
                 $event->result = $result;
             }
         }
+        $this->_dispatchedEvents[] = $event;
```

Comment:
How would you like to en/disable it? Making it a dev only feature by checking for debug = 1? Having a method like enableEventStacking() and disableEventStacking()? I'm suggesting two methods because I think in one of the other issue tickets I've seen that we want to feature getters and setters (ugly :() in the future.